.github/scripts/install-libkdumpfile.sh

```diff
@@ -10,7 +10,7 @@ git clone https://github.com/ptesarik/libkdumpfile.git
 
 cd libkdumpfile
 autoreconf -fi
-./configure
+./configure --with-python $(which python3)
```

Comment:
if you use `--with-python=$(which python3)`, I think it'll work

Reply:
Thank you for this! It does make things a lot simpler!